### DISCUSSION TITLE
Human Husks should have the ability to be disguised when the right conditions are met

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#7536] [Ideas and suggestions]
As it is now, a crew member getting converted into a husk is not to much of a threat, primarily due to the fact that their bright red name acts as a clear warning sign that this is no longer your friend. I believe that if a human husk is wearing something that covers their face, (diving mask, diving suit, clown mask, etc.) should not have their name display as red. 

Another thing is that when a player who was disguised as another player gets turned into a husk, the husks name displays as the name of the original character, which I feel doesn't make to much sense. 

I feel that from a evolutionary standpoint, a husks main advantage is the ability to disguise itself as another member of a species, which allows it to get closer to its prey and infect others. It would be cool to see this properly reflected through in game mechanics.

**Comment by Hazmat616** [comment]
I love this idea. Did you know husks that have items in their hands will still hold them when transformed? I'm not saying they should use them but having them attempt to hold stuff from inventory would help them disguise themselves and as we know husks are at least intelligent enough to use doors.
